ecosystem/sep-0009.md

```diff
@@ -16,6 +16,13 @@ This SEP defines a list of standard KYC and AML fields for use in Stellar ecosys
 
 This is a list of possible fields that may be necessary to handle many different use cases, there is no expectation that any particular fields be used for a particular application. The best fields to use in a particular case is determined by the needs of the KYC or AML application.
 
+## KYC Levels
```

Comment:
I think KYC levels are an important concept, although it's tricky since there's no universally agreed-on or standard for which pieces are information are at which level. As a result, I would advocate removing the KYC level discussion from this doc. This SEP aims to define a set of names that can be used by automated systems to refer to particular kinds of KYC information. I'd recommend that protocols based on SEP-9 refer to each piece of information that's needed separately by name, rather than grouping them together and referring to them by KYC levels. Using specific names is more precise, and avoids any confusion when entities have different definitions of the levels internally.